src/components/FilterCheckbox.tsx

```diff
@@ -0,0 +1,13 @@
+import { FormControlLabel, Checkbox } from '@mui/material';
+
+interface Props {
+  label: string
+}
+
+function FilterCheckbox({ ...props }: Props) {
```

Comment:
why are you spreading the `props` here can't you just do e.g. with Component
```js
function FilterCheckbox(props: Props) {
 return <Component {...props} /> 
}
```

Reply:
Did not notice lmao, ill commit the fix and merge.